src/unicef_security/models.py

```diff
@@ -66,10 +71,15 @@ def label(self):
             return f"{self.first_name} {self.last_name}:"
         elif self.first_name:
             return self.first_name
-        else:
-            return self.username
+        return self.username
 
     def save(self, *args, **kwargs):
         if not self.display_name:
             self.display_name = self.label
         super().save(*args, **kwargs)
+
+
+class User(ExtendedAbstractUser):
+    class Meta:
+        swappable = 'AUTH_USER_MODEL'
```

Comment:
why are we doing this?

Reply:
hey Rob I own you this!
Swappable is when you can define a model as setting, (User model in django is swappable). FK would go to the user model, this should not have a big impact.
My idea was to have a User model swappable/mixin that can be applied to our applications and we can share common functionalities (AD integration, AD login, pipelines..) and fields (azure_id, job_title.. )